modules/http4s/test/src/smithy4s/http4s/SimpleRestJsonComplianceSuite.scala

```diff
@@ -95,7 +95,8 @@ object SimpleRestJsonComplianceSuite extends ProtocolComplianceSuite {
       val baseUri = uri"http://localhost"
       val suppliedHost =
         testHost.map(host => Uri.unsafeFromString(s"http://$host"))
-      SimpleRestJsonBuilder(service)
+      SimpleRestJsonBuilder
+        .apply(service)
```

Comment:
is that a syntactic preference or did something change in a source-incompatible manner ?

Reply:
I think I probably added it temporarily so I could more easily click through to the implementation, and then forgot to revert it. I'll switch it back.